### ISSUE TITLE
Edit > Replace / Ctrl-H loses focus

### ISSUE THREAD

**Issue by brianamm** [microsoft/vscode#73347]
Issue Type: <b>Bug</b>

When using the Edit > Replace, the dialog loses focus, causing unwanted changes to file content.

Start with a text file with the following content:
one, two, three, four, five

The goal is to replace the comma and space between each word with a line break. Therefore, you want to search for ", " and replace it with "\n". 

1. Make sure the cursor is at the beginning of the line
2. Type Ctrl-H to open the Replace dialog
3. Enter (without the quotes) ", "
4. Hit the tab key to move from the "search" field to the "replace" field
5. Instead of focus moving to the next field as expected, focus returns to the file content, causing the first instance of ", " to be replaced with the value in the "replace" field

VS Code version: Code 1.33.1 (51b0b28134d51361cf996d2f0a1c698247aeabd8, 2019-04-11T08:27:14.102Z)
OS version: Windows_NT x64 10.0.17134

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Xeon(R) Silver 4110 CPU @ 2.10GHz (2 x 2095)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: disabled_off<br>native_gpu_memory_buffers: disabled_software<br>rasterization: unavailable_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|16.00GB (11.54GB free)|
|Process Argv||
|Screen Reader|no|
|VM|100%|

</details><details><summary>Extensions (21)</summary>

Extension|Author (truncated)|Version
---|---|---
npm-intellisense|chr|1.3.0
vscode-markdownlint|Dav|0.26.0
vscode-eslint|dba|1.8.2
xml|Dot|2.4.0
vscode-npm-script|eg2|0.3.5
code-runner|for|0.9.9
macros|ged|1.2.1
gc-excelviewer|Gra|2.1.32
mssql|ms-|1.6.0
python|ms-|2019.4.12954
azure-account|ms-|0.8.2
csharp|ms-|1.19.0
powershell|ms-|1.12.1
team|ms-|1.149.2
vscode-docker|Pet|0.6.2
filetemplate|Ral|2.0.4
vscode-xml|red|0.5.1
vscode-autohotkey|sle|0.2.2
code-spell-checker|str|1.7.8
sort-lines|Tyr|1.8.0
vscode-icons|vsc|8.6.0

(1 theme extensions excluded)

</details>

**Comment by rebornix** [comment]
This issue should be gone now.
